vendor/vendor.json

```diff
@@ -647,6 +647,14 @@
 			"versionExact": "v1.12.19"
 		},
 		{
+			"checksumSHA1": "3AcW4E/TxmNXfklpo1szXmY+QzE=",
+			"path": "github.com/aws/aws-sdk-go/service/glue",
+			"revision": "d46843b92a3a8e377d7fe24b4d70fca37f923d00",
+			"revisionTime": "2017-11-03T17:54:03Z",
+			"version": "v1.12.22",
+			"versionExact": "v1.12.22"
```

Comment:
The issue is that this dependency is using 1.12.22 whereas other vendors are matching 1.12.19 (in this work) or 1.12.21 is the master branch since #2169 was merged.

Could you rerun against 1.12.21? 👍

Reply:
No changes between .21 and .22 in the glue sdk for me.